src/state/routing/slice.ts

```diff
@@ -33,7 +33,7 @@ export const routing = createApi({
 
         if (
           // If enabled, try the routing API, falling back to client-side routing.
-          args.routerPreference === RouterPreference.API &&
+          (args.routerPreference === RouterPreference.API || args.routerPreference === RouterPreference.PRICE_API) &&
```

Comment:
thoughts on separating the concept of `price` query into its own argument? i feel like conflating routerPreference for both routerUrl and also use case is kinda weird

we could have `routerPreference` just reference either API | Client (and other | Gouda)

and then have a separate argument called quote type - `SwapQuote | PriceQuote | skip` and the polling interval can be determined by just that

Reply:
yeah that's fair, let me try again